Client/Clique/Modules/JoinedEvents/JoinedEvents.storyboard

```diff
@@ -11,23 +11,33 @@
         <!--Joined events-->
         <scene sceneID="s0d-6b-0kx">
             <objects>
-                <viewController id="Y6W-OH-hqX" customClass="JoinedEventsViewController" customModule="Clique" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController storyboardIdentifier="JoinedEventsView" id="Y6W-OH-hqX" customClass="JoinedEventsViewController" customModule="Clique" customModuleProvider="target" sceneMemberID="viewController">
```

Comment:
This screen is exactly the same as `CreatedEvents` - I would suggest reusing the same screen instead of doing another one.

Also, other comments apply to this design as well.